bridge/client/app/_services/_mockData/evaluations-mock.ts

```diff
@@ -0,0 +1,2760 @@
+                "violated": false
+              }
+            ]
+          }
+        ],
+        "result": "fail",
+        "score": 50,
+        "sloFileContent": "LS0tDQpzcGVjX3ZlcnNpb246ICIxLjAiDQpjb21wYXJpc29uOg0KICBhZ2dyZWdhdGVfZnVuY3Rpb246ICJhdmciDQogIGNvbXBhcmVfd2l0aDogInNpbmdsZV9yZXN1bHQiDQogIGluY2x1ZGVfcmVzdWx0X3dpdGhfc2NvcmU6ICJwYXNzIg0KICBudW1iZXJfb2ZfY29tcGFyaXNvbl9yZXN1bHRzOiAxDQpmaWx0ZXI6DQpvYmplY3RpdmVzOg0KICAtIHNsaTogInJlc3BvbnNlX3RpbWVfcDk1Ig0KICAgIGRpc3BsYXlOYW1lOiAiVGhpcyBpcyBteSB2ZXJ5IHZlcnkgdmVyeSB2ZXJ5IHZlcnkgdmVyeSB2ZXJ5IHZlcnkgdmVyeSB2ZXJ5IHZlcnkgdmVyeSBsb25nIGRpc3BsYXlOYW1lIg0KICAgIGtleV9zbGk6IGZhbHNlDQogICAgcGFzczogICAgICAgICAgICAgIyBwYXNzIGlmIChyZWxhdGl2ZSBjaGFuZ2UgPD0gMTAlIEFORCBhYnNvbHV0ZSB2YWx1ZSBpcyA8IDYwMG1zKQ0KICAgICAgLSBjcml0ZXJpYToNCiAgICAgICAgICAtICI8PSsxMCUiICAjIHJlbGF0aXZlIHZhbHVlcyByZXF1aXJlIGEgcHJlZml4ZWQgc2lnbiAocGx1cyBvciBtaW51cykNCiAgICAgICAgICAtICI8NjAwIiAgICAjIGFic29sdXRlIHZhbHVlcyBvbmx5IHJlcXVpcmUgYSBsb2dpY2FsIG9wZXJhdG9yDQogICAgd2FybmluZzogICAgICAgICAgIyBpZiB0aGUgcmVzcG9uc2UgdGltZSBpcyBiZWxvdyA4MDBtcywgdGhlIHJlc3VsdCBzaG91bGQgYmUgYSB3YXJuaW5nDQogICAgICAtIGNyaXRlcmlhOg0KICAgICAgICAgIC0gIjw9ODAwIg0KICAgIHdlaWdodDogMQ0KdG90YWxfc2NvcmU6DQogIHBhc3M6ICI5MCUiDQogIHdhcm5pbmc6ICI3NSUi",
+        "timeEnd": "2021-03-16T12:46:58Z",
+        "timeStart": "2021-03-16T12:44:54Z",
+        "sloFileContentParsed": "---\r\nspec_version: \"1.0\"\r\ncomparison:\r\n  aggregate_function: \"avg\"\r\n  compare_with: \"single_result\"\r\n  include_result_with_score: \"pass\"\r\n  number_of_comparison_results: 1\r\nfilter:\r\nobjectives:\r\n  - sli: \"response_time_p95\"\r\n    displayName: \"This is my very very very very very very very very very very very very long displayName\"\r\n    key_sli: false\r\n    pass:             # pass if (relative change <= 10% AND absolute value is < 600ms)\r\n      - criteria:\r\n          - \"<=+10%\"  # relative values require a prefixed sign (plus or minus)\r\n          - \"<600\"    # absolute values only require a logical operator\r\n    warning:          # if the response time is below 800ms, the result should be a warning\r\n      - criteria:\r\n          - \"<=800\"\r\n    weight: 1\r\ntotal_score:\r\n  pass: \"90%\"\r\n  warning: \"75%\"",
+        "score_pass": "90",
+        "score_warning": "75",
+        "compare_with": "single_result\r\n",
+        "include_result_with_score": "pass\r\n",
+        "number_of_comparison_results": 1,
+        "number_of_missing_comparison_results": 0
+      },
+      "labels": {
+        "DtCreds": "dynatrace"
+      },
+      "project": "sockshop",
+      "result": "fail",
+      "service": "carts",
+      "stage": "staging",
+      "status": "succeeded"
+    },
+    "id": "843a4a6b-4dec-4328-8579-6af3d1a6c7cb",
+    "source": "lighthouse-service",
+    "specversion": "1.0",
+    "time": "2021-03-16T12:48:05.732Z",
+    "type": "sh.keptn.event.evaluation.finished",
+    "shkeptncontext": "6f1327d2-ded2-48ab-a1c6-e4f3d0ebe432",
```

Comment:
Add this to `patterns.txt`:
```
# uuid:
[<({"'>][0-9a-fA-F]{8}-(?:[0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}[<'"})>]
```
From https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples%3A-patterns

Reply:
I added the words to `expect.txt` and the pattern to `patterns.txt`
Thanks.